src/components/AddEvent/FormsUI/TextField.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import TextField from "@mui/material/TextField";
+import { useField } from "formik";
+
+const TextFieldWrapper = ({ name, ...otherProps }) => {
+  const [field, meta] = useField(name);
+
+  const configTextField = {
+    fullWidth: true,
+    variant: "outlined",
+    ...otherProps,
+    ...field,
+  };
+
+  if (meta && meta.touched && meta.error) {
+    configTextField.error = true;
```

Comment:
from what i remember, error for fields is automatically done. Don't need to add you the error = true manually. Let's check together tomorrow